components/announcement_bar/cloud_trial_announcement_bar/cloud_trial_announcement_bar.tsx

```diff
@@ -52,11 +49,7 @@ const MAX_DAYS_BANNER = 'max_days_banner';
 const THREE_DAYS_BANNER = '3_days_banner';
 class CloudTrialAnnouncementBar extends React.PureComponent<Props> {
     async componentDidMount() {
-        if (isEmpty(this.props.analytics)) {
```

Comment:
from the very beggining seems to be that analytics were not used at all at this file, seems to have been a leftover from creating this component from another component.

Reply:
Nice find!